chpl/chpl-service/src/main/resources/errors.properties

```diff
@@ -54,6 +58,8 @@ listing.criteria.missingG2Related=(g)(2) was found without a required related ce
 listing.criteria.missingG1Related=(g)(1) was found without a required related certification.
 listing.criteria.missingComplementG3=A certification that requires (g)(3) was found but (g)(3) was not.
 listing.criteria.missingG3complement=(g)(3) was found without a required related certification.ired certification criteria 170.314 (g)(4) was not found.
+listing.criteria.missing170314g4=Required certification criteria 170.314 (g)(4) was not found.
+listing.criteria.missingUcdProccesses=Criteria % indicated SED but no UCD Processes were listed.
```

Comment:
If I'm not mistaken, the `%s` is a placeholder that means replace it as a string, not pluralize it. All of the other placeholders in this file (that I can see) have the `s` attached. Am I wrong @kekey1 ?

Reply:
Oh I see what you mean...that is the string replacement.